src/tasks/BagelAITools.py

```
@@ -0,0 +1,1157 @@
+                    else:
+                        raise Exception(f"未找到首选模型和其他视觉模型")
+
+        # ==========================================
+        # 后续的标准 Vision 请求逻辑
+        # ==========================================
+        api_url = f"{base_url}/v1/chat/completions"
+        
+        final_prompt = prompt
+        if post_title or author:
+            final_prompt += "\n\n【目标帖子信息】"
+            if post_title: final_prompt += f"\n标题: {post_title}"
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)